IORawData/SiPixelInputSources/src/PixelSLinkDataInputSource.cc

```diff
@@ -260,7 +260,7 @@ bool PixelSLinkDataInputSource::setRunAndEventInfo(edm::EventID &id,
   buffers = std::make_unique<FEDRawDataCollection>();
 
   //  uint32_t currenteventnumber = (m_data >> 32)&0x00ffffff;
-  uint32_t eventnumber = (m_data >> 32) & 0x00ffffff;
+  uint32_t eventnumber = 0;
```

Comment:
I wonder how this line can resolve a dead assignment, because the assignment is done anyhow, but with a different value. Apparently yes (I see no more warnings from the statica analyzer)... but shouldn't it be
```suggestion
  uint32_t eventnumber;
```
instead?

Reply:
I guess assigning constant value (or, maybe, only zero) is fine from LLVM analyzer's point of view. I will apply your suggestions.